HTML/Layout_1.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="layout1.css">
+    </head>
+    <body>
+        <div class="navbar">
+            <img src="images\logo.jpg" alt="logo" id="logo">
+            <button class="search1">
+                <img src="images\search.jpg" alt="search" id="search_1">
+            </button>
+        </div>
+        <div class="container1">
+            <img src="images\london.jpg" alt="london" id="london">
+            <div class="card">
+                <p>Travel the world with us</p>
+                <form class="form">
+                    <div class="depart">
+                        <label>From:</label><br>
+                        <input type="text" name="text" placeholder="Departing from">
+                    </div>
+                    <div class="arrive">
+                        <label>To:</label><br>
+                        <input type="text" name="text" placeholder="Arriving at">
+                    </div>
+                </form>
+                <button type="submit" class="search">Search and find dates</button>
+            </div>
+        </div>
+        <div class="container2">
+            <div class="text1">
+                <h3 style="font-family: Arial, Helvetica, sans-serif;">Explore Nature</h3>
```

Comment:
No inline styling unless it is absolutely necessary. Please set font family in the body.